vllm/entrypoints/openai/tool_parsers/mistral_tool_parser.py

```diff
@@ -232,7 +232,7 @@ def extract_tool_calls_streaming(
                     delta = DeltaMessage(tool_calls=[
                         DeltaToolCall(index=self.current_tool_id,
                                       type="function",
-                                      id=f"chatcmpl-tool-{random_uuid()}",
+                                      id="".join(choices(ALPHANUMERIC, k=9)),
```

Comment:
We could probably directly use the factory function here.
This would have the added benefit of being able to benefit from any update done to the id generation logic, while staying in sync with the non-stream case.
 
```suggestion
                                      id=MistralToolCall.generate_random_id(),
```